components/bl-signature-pad-component/README.md

```diff
@@ -0,0 +1,76 @@
+# Signature Pad
+
+Signature Pad is a component of Backendless UI-Builder designer. It allows to draw smooth signatures and save them for later usage. A user can pick up any color for a drawing pen. Also, the component has features for exporting signatures as images in different formats. The action buttons are configurable, and thus you can adjust their visibility and labels.
+
+The component based on external [signature_pad](https://github.com/szimek/signature_pad) library post by [Szymon Nowak](https://github.com/szimek).
+
+## Usage
+
+### API
```

Comment:
Please change headings:
'API' to 'Component Elements'
'Options' to 'Component properties'

Reply:
'Events' to 'Component Events'?